operations/reservations.md

```diff
@@ -13,11 +13,32 @@ Returns all reservations from the specified interval according to the time filte
     "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
     "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
     "Client": "Sample Client 1.0.0",
-    "GroupIds": [
-        "c704dff3-7811-4af7-a3a0-7b2b0635ac59"
-    ],
     "StartUtc": "2016-01-01T00:00:00Z",
     "EndUtc": "2016-01-07T00:00:00Z",
+    "ReservationIds": [
+        "db6cad34-9a91-448b-bea1-abbe01240d9c"
+    ],
+    "GroupIds": [
+        "4d6907df-b956-451a-81bc-ab8e00a98a68"
+    ],
+    "CustomerIds": [
+        "8e1d0ca6-1dde-4be0-8566-aafc01866110"
+    ],
+    "SpaceIds": [
+        "5ee074b1-6c86-48e8-915f-c7aa4702086f"
+    ],
+    "RateIds": [
+        "ed4b660b-19d0-434b-9360-a4de2ea42eda"
+    ],
+    "BusinessSegmentIds": [
+        "7760b5cb-a666-41bb-9758-76bf5d1df399"
+    ],
+    "Numbers": [
+        "7781"
+    ],
+    "States": [
+        "Started"
+	],
```

Comment:
Code style.

Reply:
Thank you. It's just strange that I couldn't see that in VS in the preview. Tried again with 4 spaces instead of tab.